spec/fixtures/test_chapter.playground/Contents.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+public var str = "Hello, playground"
+
+public func sharedFunc() {
+  print("This should be accessible to all pages.")
+}
+
+//// Page 1.
+import UIKit
```

Comment:
These currently need to be repeated for each page. Probably going to fix that. Maybe in `book.yml`...